app/views/site/_study_visualize.html.erb

```diff
@@ -5,7 +5,7 @@
   <div class="col-md-13" id="render-target">
     <div class="row-offcanvas row-offcanvas-right">
       <div id="view-options-nav"><a href="#view-options" id="view-option-link" data-toggle="offcanvas"><i class="fas fa-cog" aria-hidden="true"></i> View Options </a></div>
-      <ul class="nav nav-tabs" role="tablist" id="view-tabs">
+      <ul class="nav nav-tabs" role="tablist" id="view-tabs" data-tablist-name='Default'>
```

Comment:
While pairing, Eno and I arrived at:

* "Explore - Default"
* "Explore - Single gene"
* "Explore - Multiple genes"

(So using your case refinement, this would be `data-tablist-name="explore-default'`.)  This makes the "Explore" tablist names systematic -- `<parent view name>-<this view name>`, which helps support different analysis use cases.